components/api_server/tests/routes/test_auth.py

```diff
@@ -225,7 +225,7 @@ def test_login_changed_details(self, connection_mock, connection_enter):
         login_ok = {
             "ok": True,
             "email": "some_other@email.com",
-            "session_expiration_datetime": (self.NOW + timedelta(hours=24)).isoformat(),
+            "session_expiration_datetime": (self.NOW + timedelta(hours=120)).isoformat(),
```

Comment:
Should we not check the use of `USER_SESSION_DURATION`?

Reply:
When running the tests there is not necessarily a `USER_SESSION_DURATION` environment variable...